release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -124,6 +124,13 @@ For more information, see xref:../installing/installing_azure/installing-azure-c
 
 This update introduces a new option for Ingress Controllers with the `hostnetwork` endpoint strategy. You can now host multiple Ingress Controllers on the same worker node using `httpPort`, `httpsPort`, and `statsPort` binding ports.
 
+[id=ocp-4-11-multiple-configuration-control-worker-nodes]
+==== Multi-node configuration for controller and worker nodes
+
+When you configure nodes, you can use a mechanism that applies a single configuration to multiple, bare metal installer-provisioned infrastructure (IPI) nodes in a cluster. Applying a single configuration that defines multiple nodes in your cluster reduces the risk of misconfiguration due to the single-provisioning process.
```

Comment:
Suggest replacing '_...bare metal..._" with '._..bare-metal..._'. So your sentence could be something like '_...bare-metal, installer-provisioned infrastructure...'_. The [RH Supplementary Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/) suggests using the hyphenated form as an adjective. For example: "Bare-metal servers". Rest changes LGTM.

Reply:
Thanks @sayjadha ! I made this change and updated the preview.